### PR TITLE
[Auto] Localization - Translated Strings

### PR DESCRIPTION
**Pull request by csigs**

### CODE REVIEW

File: Extension/i18n/fra/package.i18n.json [hunk 1]
```diff
@@ -194,7 +194,7 @@
 	"c_cpp.debuggers.logging.trace.description": "Indicateur facultatif pour déterminer si le suivi de commande de l'adaptateur de diagnostic doit être journalisé dans la console de débogage. La valeur par défaut est false.",
 	"c_cpp.debuggers.logging.traceResponse.description": "Indicateur facultatif pour déterminer si le suivi de commande et de réponse de l'adaptateur de diagnostic doit être journalisé dans la console de débogage. La valeur par défaut est false.",
 	"c_cpp.debuggers.cppvsdbg.logging.threadExit.description": "Indicateur facultatif pour déterminer si les messages indiquant la sortie du thread doivent être journalisés dans la console de débogage. Valeur par défaut: 'false'.",
-	"c_cpp.debuggers.cppvsdbg.logging.processExit.description": "Indicateur facultatif pour déterminer si les messages indiquant la sortie du processus cible doivent être journalisés dans la console de débogage. Valeur par défaut: 'true'.",
+	"c_cpp.debuggers.cppvsdbg.logging.processExit.description": "Indicateur facultatif pour déterminer si les messages indiquant la sortie du thread doivent être journalisés dans la console de débogage. Valeur par défaut: 'false'.",
```
Comment (jogo-) [inline Extension/i18n/fra/package.i18n.json]: Regression#1

File: Extension/i18n/fra/package.i18n.json [hunk 2]
```diff
@@ -218,7 +218,7 @@
 	"c_cpp.debuggers.miDebuggerArgs.description": "Arguments supplémentaires pour le débogueur MI (par exemple gdb).",
 	"c_cpp.debuggers.miDebuggerServerAddress.description": "Adresse réseau du serveur du débogueur MI auquel se connecter (par exemple : localhost:1234).",
 	"c_cpp.debuggers.stopAtEntry.description": "Paramètre facultatif. Si la valeur est true, le débogueur doit s'arrêter au point d'entrée de la cible. Si processId est passé, le paramètre n'a aucun effet.",
-	"c_cpp.debuggers.debugServerPath.description": "Chemin complet facultatif au serveur de débogage à lancer (valeur par défaut: null). Utilisé conjointement avec \"miDebugServerAddress\" ou votre propre serveur avec \"customSetupCommand\" qui exécute \"-target-select remote <server:port>\".",
+	"c_cpp.debuggers.debugServerPath.description": "Chemin complet facultatif au serveur de débogage à lancer (valeur par défaut: null). Utilisé conjointement avec \"miDebugServerAddress\" ou votre propre serveur avec \"customSetupCommand\" qui exécute \"-target-select remote <server:port>\"`.",
```
Comment (jogo-) [inline Extension/i18n/fra/package.i18n.json]: Regression#2

File: Extension/i18n/fra/package.i18n.json (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/i18n/fra/package.i18n.json]: Yeah, we're still waiting for the translators to update the source translations.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
Don't check in unless the regressions are undone.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
Don't check this in yet -- the translations got reverted, probably should wait for the "new" translations to arrive. Also, the 2 French translations aren't fixed (@Colengms I'm on an email thread with the loc team asking about that, since it's been nearly 3 weeks).

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**